test/Microsoft.Azure.WebJobs.Host.EndToEndTests/Scale/ScaleHostEndToEndTests.cs

```diff
@@ -33,6 +33,13 @@ public class ScaleHostEndToEndTests
         [InlineData(true)]
         public async Task ScaleManager_GetScaleStatusAsync_ReturnsExpected(bool tbsEnabled)
         {
+            var triggerMetadata = new List<TriggerMetadata>()
+            {
+                new TriggerMetadata(Function1Name, new JObject { { "type", "testExtensionATrigger" } }),
+                new TriggerMetadata(Function2Name, new JObject { { "type", "testExtensionATrigger" } }),
+                new TriggerMetadata(Function3Name, new JObject { { "type", "testExtensionBTrigger" } }, new Dictionary<string, object> { { "foo", "bar" } }),
```

Comment:
I'd like to discuss more why this Properties bag is needed for triggers, but for now assuming it is, I retain it in the API.

Reply:
This is needed to pass AzureCompoentFactory for example (maybe some other properties in the future). We can not define AzureCompoentFactory in TriggerMetadata as it would bring another nuget package reference and we do no want that.